src/api/controllers/articleController.js

```diff
@@ -3,9 +3,10 @@ import models from '../models';
 import statusCodes from '../../helpers/constants/status.codes';
```

Comment:
can we switch to just ``status``? it makes more sense to see ``status.OK``
push back so I can understand your reasoning.